jquery.floatThead.js

```diff
@@ -648,10 +662,22 @@
             if (tableContainerGap > scrollingContainerTop || scrollingContainerTop - tableContainerGap > tableHeight) {
               top = tableOffset.top - windowTop;
               unfloat();
+              // if we scrolled horizontally, table size may have changed
+              // for example, if there's padding to left of table and when scroll all the way to left padding "reappears"
+              setFloatWidth(); 
               triggerFloatEvent(false);
             } else {
               top = tableOffset.top + scrollingContainerTop  - windowTop - tableContainerGap;
+              // when scrolling horizontally within a modal, size of table may have increased
+              // for example, if there's padding to left of table and when scroll padding is no longer visible
+              setFloatWidth(); 
```

Comment:
I hear the concern about the expensive method.  Any suggestions how to
solve the problem a different way?

Attached is a sample html page that illustrates the issue. Click the link
to open the modal, and then try scrolling the table both horizontally and
vertically. See attached screenshot for the result I get in Chrome.

On Fri, Aug 7, 2015 at 5:01 PM, Misha Koryak <notifications@github.com>
wrote:

> In jquery.floatThead.js
> <https://github.com/mkoryak/floatThead/pull/229#discussion_r36560878>:
>
> >                triggerFloatEvent(false);
> >              } else {
> >                top = tableOffset.top + scrollingContainerTop  - windowTop - tableContainerGap;
> > +              // when scrolling horizontally within a modal, size of table may have increased
> > +              // for example, if there's padding to left of table and when scroll padding is no longer visible
> > +              setFloatWidth();
>
> I am not sure I am ok with calling this expensive method on every scroll
> event. this will be slow for older browsers / phones
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mkoryak/floatThead/pull/229/files#r36560878>.
>